Common_Files/Physics.ipynb

```diff
@@ -10,19 +10,53 @@
     "\n",
     "**In developing CFD simulation for a fluid mechanics problem understanding the general physics and fundamental theory of the problem is extremely important. This knowledge would provide user the ability to make logical decision about the geometry of CFD domain, choice of proper boundary conditions and numerical models for the CFD simulation setup. Furthermore, user will have a general big picture of the expected results from simulation. These knowledges and abilities form the foundation for a successful and accurate implementation and possibly validation of the CFD simulation.**\n",
     "\n",
-    "**In other words understanding physics and theory of problem before developing a CFD simulation can be thought of turning on a flash light prior to taking any step in a completely dark room. Hence, let's review the fundamental physics and theory behind the problem of \"[insert the topic of problem here]\" as the first step for development and validation of CFD simulation for the problem of < insert the title of the problem > :**\n",
+    "**In other words understanding physics and theory of problem before developing a CFD simulation can be thought of turning on a flash light prior to taking any step in a completely dark room. Hence, let's review the fundamental physics and theory behind the problem of \"2D Laminar Flow over a rectangular plate.\" as the first step for development and validation of CFD simulation for the problem of < insert the title of the problem > :**\n",
     "\n",
     "# General Recommended Structure for Writing Physics File\n",
     "\n",
-    "Below is the recommended general structure to develop the Physics file for the flow of interest. Please note that we are always open to new ideas and modifying the structure below. Thus, do not limit yourself to the following structure and feel free to implement your suggestion and changes. We can modify this structure based on new feedback.\n",
+    "<img src=\"./Images/fluid1.jpg\" width=\"500\">\n",
+    "\n",
```

Comment:
Add a general and descriptive explanation of the physics behind the steady and uniform flow over a rectangle.  Use/Create a simple "sketch" instead of the colorful CFD results. Note that your readers are learning CFD and have not idea at the Physics section about it.